assets/js/components/user-input/UserInputKeywords.js

```diff
@@ -113,6 +113,12 @@ export default function UserInputKeywords( { slug, max } ) {
 	return (
 		<Cell lgStart={ 6 } lgSize={ 6 } mdSize={ 8 } smSize={ 4 }>
 			<div ref={ keywordsContainer } className="googlesitekit-user-input__text-options">
+				<label
+					htmlFor={ `${ slug }-keyword-0` }
```

Comment:
This will always generate a label for the first keyword, which means this will be a label linked to no element after the first keyword is entered. This label should be generated inside the `values.map` next to the `<TextField><Input /></TextField>` it is for, because the ID changes based on the index.

Also, we have a `VisuallyHidden` component I'd prefer we use over the CSS class. Building as many things out of components is more React-y. Admittedly it's sort of a preference, but it's why we have the `VisuallyHidden` component. So let's use that instead 🙂

Reply:
@tofumatt if we want to generate that label for every keyword, we need to update label text because otherwise screen readers will read the same label text triple times which won't help users with disabilities too much.

Reply:
True, let's do that. Having only one and linking it with a specific index would mean the other inputs don't have useful labels still though.

That or we change the `htmlFor` based on the input, but having a label for each strikes me as better…